File: Employee_Directory.Models/EmployeeCard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Employee_Directory.Models
{
    public class EmployeeCard
    {
        public int Id { get; set; }

        public string  PreferredName { get; set; }

        public string Department { get; set; }

        public string  JobTitle { get; set; }

        public string Office { get; set; }
    }
}

File: Employee Directory/ClientApp/src/app/models/EmployeeCard.ts
export class EmployeeCard {
  public id: number;
  public preferredName: string;
  public email: string;
  public department: number;
  public office: number;
  public jobTitle: number;

  constructor(args: any) {
    args = !!args ? args : {};
    this.id = args.id;
    this.preferredName = args.preferredName;
    this.email = args.Email;
    this.department = args.department;
    this.office = args.office;
    this.jobTitle = args.jobTitle;
  }
}
